setup.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+from pip.req import parse_requirements
+import setuptools
+
+# Filters out relative/local requirements (i.e. ../lib/utils)
+remote_requirements = '\n'.join(str(r.req) for r in parse_requirements("requirements.txt", session='dummy') if r.req)
+
+setuptools.setup(
+    name='kafka-streams-examples-tests',
+    version='0.0.1',
```

Comment:
can we make this just match the version in the pom? in fact, can we just get the version from the pom?

Reply:
The POM version is inherited from the parent -- we could just grab the parent version, but it's still pretty hacky.

I think we can just hard-code it to match the version for now.